### PR TITLE
Improve PageUp/PageDown keybinds for Linux/Mac

### LINKED ISSUE #13389: Move cursor when pressing PageUp or PageDown
### Check for existing issues

- [X] Completed

### Describe the feature

In VSCode using the "PageUp" or "PageDown" keys moves also moves the cursor up and down. 
Zed does not do this and it is very annoying. When quickly scanning a file to find a relavent line I then have to nevigate there again with the arrow-keys (and have to find it again).
Especially on Thinkpad-like keyboards (where PageUp/Down keys are directly above the arrow-keys this a way of navigating files quickly that I (and presumably) many people use _constantly_.

### If applicable, add mockups / screenshots to help present your vision of the feature

_No response_

### PR DESCRIPTION
**Pull request by notpeter**
Improves PageUp/PageDown behavior to match VSCode [macOS shortcuts](https://go.microsoft.com/fwlink/?linkid=832143) and [Linux shortcuts](https://go.microsoft.com/fwlink/?linkid=832144).

Fixes: #13389.

Notably this makes pageup/pagedown move the cursor. 

Release Notes:

- Changed the default key bindings for pageup/pageup on Mac and Linux so that it moves the cursor (`editor::MovePageDown`). Previous scroll only behavior (`editor::PageDown`) is now available via `cmd-pagedown` (MacOS) and `alt-pagedown` (Linux).

### CODE REVIEW

**Comment by maxdeviant** [comment]
@notpeter Do we need to add release notes for this? Seems like it's a user-facing change.

**Comment by notpeter** [comment]
> @notpeter Do we need to add release notes for this? Seems like it's a user-facing change.

Done.

File: assets/keymaps/default-linux.json [hunk 1]
```diff
@@ -55,11 +55,13 @@
       "up": "editor::MoveUp",
       "ctrl-up": "editor::LineUp",
       "ctrl-down": "editor::LineDown",
-      "pageup": "editor::PageUp",
+      "pageup": "editor::MovePageDown",
```
Comment (llogick) [inline assets/keymaps/default-linux.json:58]:
```suggestion
      "pageup": "editor::MovePageUp",
```

Typo

Comment (notpeter) [reply assets/keymaps/default-linux.json]: Good catch! Thanks

**Pull request merged**